packages/react/src/hooks/useTranscriptions.ts

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useTextStream } from './useTextStream';
+import { DataTopic } from '@livekit/components-core';
+
+/**
+ * @beta
+ */
+export interface UseTranscriptionsOptions {
+  participantIdentities?: string[];
+  trackSids?: string[];
+}
+
+/**
+ * @beta
+ * useTranscriptions is a hook that returns the transcriptions for the given participant identities and track sids,
+ * if no options are provided, it will return all transcriptions
+ * @example
+ * ```tsx
+ * const transcriptions = useTranscriptions();
+ * return <div>{transcriptions.map((transcription) => transcription.text)}</div>;
+ * ```
+ */
+export function useTranscriptions(opts?: UseTranscriptionsOptions) {
+  const { participantIdentities, trackSids } = opts ?? {};
```

Comment:
doesn't adding these options expose an easy trap where you have

```
const agentTranscriptions = useTranscriptions({ participantIdentities: ['agent'] });
```

and

```
const userTranscriptions = useTranscriptions({ participantIdentities: ['user'] });
```

in two different locations and then they conflict on `useTextStream` which can only be used once?

Reply:
ah ok that works. i do still think we should just make it easy for anyone to register multiple handlers but it's not a blocker to this PR